DUI3-DX/Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SendBinding.cs

```diff
@@ -39,7 +39,7 @@ IBridge bridge
     _speckleConverterToSpeckleFactory = speckleConverterToSpeckleFactory;
     _speckleConverterToSpeckle = _speckleConverterToSpeckleFactory.ResolveScopedInstance();
     _idleManager = idleManager;
-
+    Commands = new SendBindingUICommands(bridge);
```

Comment:
yeah we're going to need to inject this ideally but it could wait until we get some conversions in